hw2/05-form.html

```diff
@@ -7,13 +7,82 @@
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
-      <form>
+  <body class="bg-dark">
+    <main class="row justify-content-center">
+      <form id="userForm" class="bg-light mt-5 w-75 p-3">
         <h1>Exercise 05 - Form</h1>
+        <label for="name" class="fw-bold">Full Name:</label> <br />
+        <input type="text" id="name" class="w-100" />
+        <label for="email" class="fw-bold">Email:</label> <br />
+        <input type="email" id="email" class="w-100" />
+        <label for="regStatus" class="fw-bold">Registration Status:</label>
+        <br />
+        <select name="regStatus" id="regStatus" class="w-100">
+          <option value="" class="fw-bold">Choose an Option</option>
+          <option value="registered">Registered</option>
+          <option value="pending">Pending</option>
+          <option value="waitlist">Waitlisted</option>
+          <option value="notRegistered">Not Registered</option>
+          <option value="other">Other</option>
+        </select>
+        <fieldset>
+          <legend class="fw-bold">
+            Have you taken any of the following courses?
+          </legend>
+          <div>
+            <input type="checkbox" id="progLangs" name="progLangs" />
+            <label for="progLangs">Programming Languages</label>
+          </div>
+          <div>
+            <input type="checkbox" id="os" name="os" />
+            <label for="os">Operating Systems</label>
+          </div>
+          <div>
+            <input type="checkbox" id="fullstack" name="fullstack" />
+            <label for="fullstack">Full Stack Web Development</label>
+          </div>
+        </fieldset>
+        <label for="also" class="fw-bold">Anything else?</label> <br />
+        <input type="text" id="also" class="w-100 h-auto" />
+        <input
+          type="submit"
+          id="submitButton"
+          value="Submit"
+          data-toggle="modal"
+          data-target="#responseModal"
+          class="btn btn-primary col-5 m-2"
+        />
+        <input type="reset" class="btn btn-secondary col-5 m-2" value="Reset" />
+
+        <div
+          class="modal fade"
+          role="dialog"
+          id="responseModal"
+          aria-labelledby="modalLabel"
+          aria-hidden="true"
+          data-bs-backdrop="true"
+        >
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title" id="modalLabel">Your responses</h5>
+                <button
+                  type="button"
+                  class="close"
+                  data-dismiss="modal"
+                  aria-label="Close"
+                ></button>
+              </div>
+              <div class="modal-body">
+                <p>Modal body (placeholder)</p>
+              </div>
+            </div>
+          </div>
+        </div>
       </form>
     </main>
```

Comment:
Devam mentioned trying to add a certain script to fix your modal error.

I believe that script is supposed to go right here, right above the closing body element tag.

Reply:
Yep, that worked, thanks!